Gemfile

```diff
@@ -1,54 +1,39 @@
+# frozen_string_literal: true
+
 source 'https://rubygems.org'
 
 git_source(:github) do |repo_name|
```

Comment:
Awesome, I was able to remove that. I added guard-rubocop and started a docs directory with DEVELOPMENT.md that we can link to from the README. 

Nudging @calvinfung re: docs :1st_place_medal: